src/clr-angular/forms/styles/containers.spec.ts

```diff
@@ -277,7 +277,7 @@ describe('Form layouts', () => {
     const computed = height(selector);
     expect(Math.round(computed)).toEqual(Math.round(value));
     if (verticalRhythm) {
-      expect(value % base).toEqual(0);
+      expect(value % 6).toEqual(0);
```

Comment:
Can you revert this line? `base` should be 6 and was helpful to communicate the meaning.